lib/device_api/android/plugins/battery.rb

```diff
@@ -0,0 +1,29 @@
+module DeviceAPI
+  module Android
+    module Plugin
+	class Battery
+   
+   	attr_accessor :battery
+	def initialize(options = {})
+		@battery = {}
+		props = {}
+		serial = options[:serial]
+		info = options[:data] || ADB.dumpsys(serial, 'battery')
+		info.each do |i|
+			key = i.split(": ")[0].gsub(" ","_")
+			props[key] = "#{i.split(": ")[1]}"
+		end
```

Comment:
Lines 11 - 15 can be replaced with props = ADB.get_battery_info(serial) as that already handles changing the ADB output into a hash.